### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by TreephobKho**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/TreephobFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class TreephobFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (TreephobKho) [inline lab-exercise-1/TreephobFeature.java:3]: Please include this part.

Comment (ACitronella) [inline lab-exercise-1/TreephobFeature.java:3]: pls add functionality

Comment (Pakin-aw) [inline lab-exercise-1/TreephobFeature.java:3]: Nice comment

File: lab-exercise-1/TreephobFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+      }
```
Comment (BunradarCH) [inline lab-exercise-1/TreephobFeature.java:4]: good feature

File: lab-exercise-1/TreephobFeature.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+  public class TreephobFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (sorrymaker1130) [inline lab-exercise-1/TreephobFeature.java:3]: Nice job

File: lab-exercise-1/TreephobFeature.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+		  int x = 0;
+          x = x+1;
+      }
```
Comment (TreephobKho) [inline lab-exercise-1/TreephobFeature.java:6]: Thank you for the edit.

File: lab-exercise-1/TreephobFeature.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+  public class TreephobFeature {
+      public void feature() {
+          // this is a new feature
+		  int x = 0;
+          x = x+1;
```
Comment (ACitronella) [inline lab-exercise-1/TreephobFeature.java:5]: outstanding change

Comment (Pakin-aw) [inline lab-exercise-1/TreephobFeature.java:5]: Nice implementation.

File: lab-exercise-1/TreephobFeature.java (hunk 3 above)
Comment (RolinAnn) [inline lab-exercise-1/TreephobFeature.java:3]: Nice feature.

File: lab-exercise-1/TreephobFeature.java (hunk 5 above)
Comment (sorrymaker1130) [inline lab-exercise-1/TreephobFeature.java:5]: Add the part successful.

Comment (RolinAnn) [inline lab-exercise-1/TreephobFeature.java:5]: Thank you for adding this part.

File: lab-exercise-1/TreephobFeature.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+  public class TreephobFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/TreephobFeature.java:1]: good

**Pull request merged**